Models/Category.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace TimeMgmt.Models
+{
+    public class Category
+    {
+        [Key]
+        [Required]
+        public int categoryid { get; set; }
```

Comment:
for consistency, can this be changed to CategoryID and CategoryName? In .NET, properties are typically capitalized.